### PR TITLE
Added ellipsis in card

### PR DESCRIPTION
**Pull request by NishantSinghhhhh**
Description
This CSS update enforces a multi-line ellipsis on text within .card .text, ensuring it displays up to a specified number of lines (i.e. 3) before truncating. The ellipsis will appear at the end of the last visible line, hiding any content that extends beyond this point.

Details

1). Sets display: -webkit-box with -webkit-line-clamp: 3 to limit text display to three lines.
2). Adds overflow: hidden and text-overflow: ellipsis to manage overflow and apply an ellipsis (...) at the end of the visible lines.

==> Notes for Reviewers
1). Adjust the line-height and -webkit-line-clamp values as needed for different line limits.
2). Ensure compatibility with WebKit browsers, as -webkit-line-clamp may not work in non-WebKit environments.

Screenshot 2024-11-02 122201

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit db51dab270d8dcc67d93d38f42ba4781a212db93 at: https://6725cf10d4550a43be1423c7--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e1f1bd9595052126467c1ee3966c0057dbd5f756 at: https://6726555a2a603eba4f801e1a--layer5.netlify.app

File: src/sections/Projects/Sistent/sistent.style.js [hunk 1]
```diff
@@ -645,13 +645,15 @@ const SistentWrapper = styled.div`
 
   .product_cards .card {
     width: 400px;
+    height: 300px;
```
Comment (sudhanshutech) [inline src/sections/Projects/Sistent/sistent.style.js:648]: will you make it little less like 280 px

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Review by sudhanshutech — COMMENTED** [review]
@NishantSinghhhhh have you checked these changes not effecting badly somewhere else if this card would have been reused somewhere?

File: src/sections/Projects/Sistent/sistent.style.js (hunk 1 above)
Comment (NishantSinghhhhh) [reply src/sections/Projects/Sistent/sistent.style.js:648]: Okay sir sure

**Comment by vishalvivekm** [comment]
@NishantSinghhhhh Let's discuss this on websites call on Monday at 6:30 PM IST (7:00 AM CT). Add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk), if you would :)

**Comment by NishantSinghhhhh** [comment]
Sure Sir

**Comment by NishantSinghhhhh** [comment]
Sure Sir

**Comment by NishantSinghhhhh** [comment]
> @NishantSinghhhhh have you checked these changes not effecting badly somewhere else if this card would have been reused somewhere?

Yes sir , I just did this, it was working fine

**Comment by l5io** [comment]
🚀 Preview for commit 8de89d2f2a3c2af830fa81434d4ade5529ed6c94 at: https://67279ec6388ea8539a5763df--layer5.netlify.app

**Review by sudhanshutech — APPROVED** [review]

**Pull request merged**